app-engine/image-resizing/main.py

```diff
@@ -50,14 +51,22 @@ def root():
         abort(404)
 
     blobstore_filename = "/gs/{}".format(filename)
+    if random.randint(1, 1000) == 1:
+        logging.info("Failed getting serving url on filename %s", blobstore_filename)
```

Comment:
Euh, on devrait peut-être éviter de logger un message d'erreur (même au niveau INFO) quand on n'a pas d'erreur, non ?

```suggestion
        logging.info("Trying to serve %s", blobstore_filename)
```

Reply:
euh oui, my bad